cognite/neat/graph/stores/_graphdb_store.py

```diff
@@ -47,5 +47,5 @@ def _set_graph(self) -> None:
 
     def drop(self):
         """Drops the graph."""
-        r = requests.delete(f"{self.graph_db_rest_url}/repositories/{self.graph_name}/rdf-graphs/service?default")
```

Comment:
This is the only place `graph_db_rest_usl` is used, can you double check and if it is, delete the attribute.

Reply:
What about here? 
https://github.com/cognitedata/neat/blob/main/cognite/neat/workflows/steps/lib/graph_store.py#L131